docs/docs/default-actions.mdx

```diff
@@ -187,3 +187,52 @@ Overriding the default action for forms should **only** be used during the proce
 migrating from Rasa Open Source 1 to 2. In this case you can override the default
 action to instruct Rasa Open Source to use the deprecated `FormAction` which is part of
 the Rasa SDK.
+
+## action_unlikely_intent
+
+Rasa Open Source triggers `action_unlikely_intent` via [`UnexpecTEDIntentPolicy`](./policies.mdx#unexpected-intent-policy).
+You can control how often this action is predicted by tuning the [`tolerance`](./policies.mdx#unexpected-intent-policy)
+parameter of `UnexpecTEDIntentPolicy`.
+
+### Customization
+
+You can customize your assistant's behaviour to configure what should happen once `action_unlikely_intent`
+is triggered. For example, as a follow up you can trigger a hand-off to a human agent with a rule:
+
+```yaml
+- rule: trigger human handoff with action_unlikely_intent
+  steps:
+    - action: action_unlikely_intent
+    - action: ask_human_handoff
+    - intent: affirm
+    - action: trigger_human_handoff
+```
+
+Alternatively, you can also override it's behaviour as a [`custom action`](./custom-actions.mdx):
```

Comment:
I think we should mention here that if you want to customise the action, you have to add `action_unlikely_intent` to the action list in the domain.